DOCS-7227-changed the title and edited the page (#211)
mulesoft/docs-api-manager: modules/ROOT/pages/import-api-task.adoc

= Creating an API Instance by Importing to API Manager
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> You might have developed an API and exported it from API Manager to share it across your organization or to use it at a later time. You can then import this API into API Manager when required. 

Reviewer: ```suggestion
Imagine that you develop an API that you export from API Manager so that you can share it across your organization or use it later. You can still import this API back into API Manager, as required. 
```
I'm not sure that I understood this correctly. 
Is the point that we are trying to make that after you create an API in API Manager, you can export it to be used elsewhere, but you can also later retrieve it? 
I get confused at "or to use it at a later time"; what does that mean, that you export it and save it somewhere outside of API Manager? Why would you do that? Under what circumstances?
And then why would you have to return it to API Manager?
So, this topic isn't really about creating by import, but about returning from export?
Consider running this intro para by another writer and see what they have to offer about clarifying the wording, because I don't understand what you are trying to say.

Author: Fixed as I saw fit. Thanks!

> == Task Prerequisite

Reviewer: You should not use this subhead, because there is not a peer subhead for the actual task, so it looks like the task is part of the Task Prerequisite text.
```suggestion
```

Author: Done! Thanks!

> Before you import an API to API Manager, you must have already exported the API instance ZIP file to you file system on your local machine or a server.

Reviewer: Is the audience RETURNING an API to API Manager, or importing a "foreign" API?
Where is the ZIP file being exported FROM?
Does this mean the same thing as "Before you import...to API Manager, you must import to your local file system"?
```suggestion
Before you can import an API to API Manager, you must have exported the API's ZIP file to your local file system or server.
```

Author: Done! Thanks!

> To import the API instance to API Manager:

Reviewer: So this is the actual task, not a prereq., so it you keep the prereq title, then you need to insert a title like "Import an API Instance to API Manager."
However, why is this worded differently from the main title: CREATE an API by IMPORTING...?
Create and import are two different things.

Author: Done! Thanks!

. Go to *API Manager > API Administration*.

> . From the main page, select *Manage API > Import*.

Reviewer: What is the main page?
Why specify this, if this is the location reached in Step 1?
```suggestion
. Select *Manage API > Import*.
```

Author: Done! Thanks

> . In the file browser, select the API instance ZIP file to import.

Reviewer: How did the user get to a browser?
This term might be misleading.
Can you use only...:
```suggestion
. Select the ZIP file of the API that you want to import.
```

Author: Added it to the previous step explained.

Author: Done! Thanks!
